zombies.js

```diff
@@ -240,13 +319,18 @@
  * @param {number} strength         The zombie's strength.
  * @param {number} speed            The zombie's speed.
  */
-
-
 /**
  * FastZombie Extends Zombie Class
  * -----------------------------
  */
-
+function FastZombie (health, strength, speed) {
+  this.health = health;
```

Comment:
For all children zombie prototypes, the health, speed, strength, and maxHealth properties do not need to be set in the constructor as you should be using the properties defined by the parent prototype.